src/main/java/com/google/ehub/servlets/ItemSubmissionServlet.java

```diff
@@ -25,16 +26,15 @@
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String title = request.getParameter(TITLE_PARAMETER_KEY);
     String description = request.getParameter(DESCRIPTION_PARAMETER_KEY);
-    Optional<String> imageURLOptional =
-        BlobstoreURLUtility.getUploadURL(request, IMAGE_URL_PARAMETER_KEY);
+    Optional<String> imageURL = BlobstoreURLUtility.getUploadURL(request, IMAGE_URL_PARAMETER_KEY);
 
-    if (!arePostRequestParametersValid(title, description, imageURLOptional)) {
+    if (!arePostRequestParametersValid(title, description, imageURL)) {
       System.err.println("ItemSubmissionServlet: Post Request parameters not specified!");
       return;
     }
 
-    EntertainmentItemDatastore.addEntertainmentItemToDatastore(
-        title, description, imageURLOptional.get());
+    EntertainmentItemDatastore.getInstance().addItemToDatastore(
+        new EntertainmentItem(/* Unassigned ID */ -1, title, description, imageURL.get()));
```

Comment:
-1 seems a little sketchy to me. One fix that comes to mind is creating a static constructor like:

...addItemToDatastore(EntertainmentItem.unassignedItem(title, description, url))

and that method can generate an id

(let's create an issue and add a TODO and fix this as a seprate PR)

Reply:
OK I created an issue and will address this in a new PR.

Reply:
can you also add a TODO here?

`// TODO(bryantriana): Fix ...`